kernel/Makefile

```diff
@@ -8,7 +8,7 @@ CFLAGS += -pipe -std=gnu99 -ffreestanding -Wall -Werror -g -O0 -mcpu=arm1176jzf-
 #PI_CFLAGS = -mfpu=vfp -march=armv6zk -mtune=arm1176jzf-s -nostartfiles
 
 # variables to define in the preprocessor.
-MEMORY = 1G
+MEMORY = 512M
```

Comment:
We probably want to change this back to 1gig before merging right?

Reply:
I thought I already changed that back, but we can leave it at 512M if we actually give this info to the mmu.

Reply:
changed it back for now

Reply:
but more ram more better right? (it likely doesn't matter that much tbf)